backend/app/api/routes/organizations.py

```
@@ -0,0 +1,45 @@
+):
+    if payload["role"] != "ADMIN":
+        raise HTTPException(status_code=403, detail="Only admins can create orgs")
+
+    org_obj = models.organization.Organization(name=org.name, created_by=payload["sub"])
+    db.add(org_obj)
+    db.commit()
+    db.refresh(org_obj)
+    return org_obj
+
+@router.get("/", response_model=list[schemas.organization.OrganizationOut])
+def list_orgs(payload: dict = Depends(get_current_payload), db: Session = Depends(get_db)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)